Project.toml

```diff
@@ -28,6 +29,8 @@ NonconvexPercival = "4296f080-e499-44ba-a02c-ae40015c44d5"
 NonconvexSemidefinite = "9e21ff56-eb25-42d3-b86f-5b0612f555e7"
 Parameters = "d96e819e-fc66-5662-9728-84c9c7592b0a"
 Pkg = "44cfe95a-1eb2-52ea-b672-e2afdf69b78f"
+PlotlyJS = "f0f68f2c-4968-5e81-91da-67840de0976a"
+Plots = "91a5bcdd-55d7-5caf-9e0b-520d859cae80"
```

Comment:
Let's consider using Makie if possible since we already have an extension for Makie.